test/puppeteer/index.test.ts

```diff
@@ -0,0 +1,15 @@
+// Imports
+import path from 'path';
+
+// Tests
+beforeEach(async () => {
+	await jestPuppeteer.resetPage();
+
+	await page.goto(`file://${path.join(__dirname, 'index.html')}`, {
+		waitUntil: 'load',
+	});
+});
+
```

Comment:
And then you should add this here:

```suggestion

// Tests
```